src/main/kotlin/ChattORE.kt

```diff
@@ -34,32 +39,35 @@ class ChattORE : Plugin() {
             setDefaultExceptionHandler(::handleCommandException, false)
         }
         discordMap = loadDiscordTokens()
+        discordMap.forEach { (_, discordApi) -> discordApi.updateActivity(config[ChattORESpec.discord.playingMessage]) }
+        discordMap.values.firstOrNull()!!.addListener(DiscordListener(this))
```

Comment:
```suggestion
        discordMap.values.first().addListener(DiscordListener(this))
```
Throws up with a bit better message if it fails